src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,368 @@
+                    "title": "MPRA Library Complexity",
+                    "description": "The complexity is the number of unique constructs, or total element-barcode pairs in the library.",
+                    "type": "number",
+                    "minimum": 0
+                }
+            }
+        },
+        "crispr_library_details": {
+            "title": "CRISPR Construct Library Details",
+            "description": "Details about a CRISPR construct library.",
+            "type": "object",
+            "additionalProperties": false,
+            "required": [
+                "guide_type"
+            ],
+            "dependentSchemas": {
+                "average_guide_coverage": {
+                    "not": {
+                        "required": [
+                            "guide_coverage_range"
+                        ]
+                    },
+                    "comment": "Specification of average_guide_coverage is mutually exclusive with specification of guide_coverage_range."
+                },
+                "guide_coverage_range": {
+                    "not": {
+                        "required": [
+                            "average_guide_coverage"
+                        ]
+                    },
+                    "comment": "Specification of guide_coverage_range is mutually exclusive with specification of average_guide_coverage."
+                }
+            },
+            "properties": {
+                "guide_type": {
+                    "title": "Guide Type",
+                    "description": "The design of guides used in a CRISPR library, paired-guide (pgRNA) or single-guide (sgRNA).",
+                    "type": "string",
+                    "enum": [
+                        "sgRNA",
+                        "pgRNA"
+                    ]
+                },
+                "tiling_modality": {
+                    "title": "Tiling Modality",
+                    "description": "The tiling modality of guides across elements or loci in a CRISPR library.",
+                    "comment": "Peak tiling = all possible gRNA in a selection of peaks. Full tiling = all possible gRNA across an entire locus, including regulatory elements and non regulatory elements. Sparse peaks = a small set of gRNA against a selection of peaks.",
+                    "type": "string",
+                    "enum": [
+                        "peak tiling",
+                        "full tiling",
+                        "sparse peaks"
+                    ]
+                },
+                "average_guide_coverage": {
+                    "title": "Average Guide Coverage",
+                    "description": "The average number of guides targeting each element of interest in the library.",
+                    "comment": "May not be submitted with guide_coverage_range.",
+                    "type": "number",
+                    "minimum": 0
+                },
+                "guide_coverage_range": {
```

Comment:
I suggest to convert into min and max in the range, otherwise it is not numeric and is difficult to interpret/use

Reply:
Done, did this with fragment size as well